src/utils/transaction/types.ts

```diff
@@ -41,6 +41,7 @@ export type InsertedTransaction = {
   // User Operation
   signerAddress?: Address;
   accountAddress?: Address;
+  accountSalt?: string;
```

Comment:
Maybe use `Hex` here? You may need to type it `as Hex` when inserting it but it's prob more correct.

Reply:
same as above, I actually want ppl to pass us strings without having to worry about hexing.

updated the header description

Reply:
I see, it may not be hexified yet when it gets stored here.